src/Networks/Blockcore.Networks.XRC/Consensus/XRCBlockHeader.cs

```diff
@@ -213,6 +217,66 @@ public Target GetWorkRequiredDigiShield(ChainedHeader chainedHeaderToValidate, X
             return finalTarget;
         }
 
+        public Target GetWorkRequiredDarkGravityWave(ChainedHeader chainedHeaderToValidate, XRCConsensus consensus)
+        {
+            var nAveragingInterval = 75; // block
```

Comment:
We may need to fine-tune the number of blocks in the averaging interval using experiments

Reply:
You can do a blockchain restore on the test network. So for testing it is better to do compilation only at server level without GIT.

Repeat:
1) Set the value of nAveragingInterval 
2) Compile it on the server and run node
3) Mining
4) Write the result
5) Reset the chain
6) do the same with a different nAveragingInterval value

I spent about a month and a half testing X11 and DigiShield. 
The problem of overloading the mining force did not occur to me and I had no way to test it.

Reply:
Thanks. I will change nAveragingInterval to 24 to begin with. If you think of any other scenarios to check, we can do either live tests or python simulations. The python simulations will not be exactly the same as real life, but should be good enough to have an idea the how the function will behave.